wfl/Models/FoodOrder.cs

```diff
@@ -46,4 +46,9 @@ public void addMenuSelection(MenuSelection selection)
             }
         }
     }
+
+    public class StatusOptionsRepresentation
```

Comment:
Yes...since we have a finite list of statuses, I explained how with hypermedia we could expose that to the user so they know the list of valid statuses to query on since it's not a criteria field...